### DISCUSSION TITLE
[M-Z] (22 Gennaio 2019) Esercizio 7

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#494] [Esami & Esoneri M-Z]
### Traccia

<img width="868" alt="image" src="https://github.com/user-attachments/assets/b3e3473c-40b4-4158-bb40-2c13d86faa06">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190122.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190122.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/122b6d8d-5aaa-4531-9f10-9577827a357f)
